### PR TITLE
forget password page

### LINKED ISSUE #217: Forget Password Page
### **Describe the bug:**

There is option of forget password in sign in page but no forget page . I want to work on this issue under GSSoC'21 Participant.

### PR DESCRIPTION
**Pull request by kumarishalini6**
# Description

I have made a forget password page and add navbar in the top.
 Fixed #217

## Type of change

Opportunity Calendar - Personal - Microsoft​ Edge 4_17_2021 10_36_24 PM
Opportunity Calendar - Personal - Microsoft​ Edge 4_17_2021 10_37_00 PM

### CODE REVIEW

**Review by vaishali614 — CHANGES_REQUESTED** [review]
@kumarishalini6 There are a few things to be changed -
1. The card looks too big, please keep it of same size as that of signup and login page.
2. Please add an image following the color scheme of the website.
3. There shouldn't be a 'Sign in with Google' button on a Forget password page.

**Comment by kumarishalini6** [comment]
Ok I'll do that

**Comment by kumarishalini6** [comment]
now the size and image are 
Opportunity Calendar - Personal - Microsoft​ Edge 4_19_2021 07_02_29 PM
ok?

**Comment by kumarishalini6** [comment]
Opportunity Calendar - Personal - Microsoft​ Edge 4_19_2021 07_10_23 PM

**Comment by kumarishalini6** [comment]
please review this PR. I want t work on another issue.

**Comment by vaishali614** [comment]
> now the size and image are
> Opportunity Calendar - Personal - Microsoft​ Edge 4_19_2021 07_02_29 PM
> ok?

1) The image still doesn't follow the color scheme of website. We're using #008DC8. Please update the image with this color code.
2) Font size is too big. Please keep it consistent with the Signup and Login forms.
3) You can remove **Regenerate Password**

**Comment by kumarishalini6** [comment]
ok

**Comment by kumarishalini6** [comment]
what about this image
man-forgot-password-concept-computer-unhappy-businessman-near-computer-forgot-account-login-password-vector-male-character-158893967

**Comment by kumarishalini6** [comment]
I guess this one is follow the website color.

**Comment by vaishali614** [comment]
@kumarishalini6 Please use the color code #008DC8 to verify the color.

**Comment by kumarishalini6** [comment]
I have exam in this week after that I'll work.

**Comment by kumarishalini6** [comment]
self-service-password-reset-user-icon-png-favpng-5xmkeEBYK8s4fUXss089AbqcS
#008dc8 - Google Search - Google Chrome 4_25_2021 10_54_55 AM
13_Laptop_reset_password_password-512

**Comment by kumarishalini6** [comment]
please check this images .

**Comment by ViVa98** [comment]
Forgot password-rafiki

Forgot password-bro

Forgot password-pana


 add the text "Illustration by Freepik Storyset" in alt or somewhere.

**Comment by kumarishalini6** [comment]
Can I ask you something?
How didi you change the color ??
And I'll change the text

**Comment by ViVa98** [comment]
I think you got those previous images from a direct google search. When I saw the image used I thought that the image is shown from "freepik" as I use mostly as my design resources. Once you select any image wanting to use, there is an edit option that redirects you to the online editor where you can edit a particular image.

**Comment by kumarishalini6** [comment]
Thanks , I didn't know about it , now it helps me a lot thanks once again

**Comment by kumarishalini6** [comment]
Now check the PR

**Comment by kumarishalini6** [comment]
Opportunity Calendar - Personal - Microsoft​ Edge 4_25_2021 06_31_58 PM

**Comment by vaishali614** [comment]
Image is perfect now. But the form seems too big as compared to the login/signup forms. Please refer the images below - 

Screenshot (428)
Screenshot (429)

I think padding of input fields can be reduced and do check for the font-sizes. Also, please change the heading to 'Forgot Password' instead of 'Forget Password'.

Rest everything seems good.

File: src/Components/SignupLogin/Login.js [hunk 1]
```diff
@@ -11,64 +11,44 @@ const Login = () => {
     setShowPassword((prevShowPassword) => !prevShowPassword);
 
   return (
-    <div>
```
Comment (vaishali614) [inline src/Components/SignupLogin/Login.js]: @kumarishalini6 Why have you edited this file and removed Navbar? Please make sure to not edit a file which is not concerned with the assigned issue.

**Review by vaishali614 — CHANGES_REQUESTED** [review]

**Comment by vaishali614** [comment]
Also, I noticed that your changes have changed the color of our homepage banner.

Capture

This is the original color:
Capture2

Please check and revert any such change.

**Comment by kumarishalini6** [comment]
Sorry about sign in page it happens by mistake
and i have done all the changes 
Opportunity Calendar - Personal - Microsoft​ Edge 4_26_2021 06_40_07 PM

**Review by Manvityagi — APPROVED** [review]

**Review by vaishali614 — APPROVED** [review]

**Comment by vaishali614** [comment]
@kumarishalini6 Changes look good now, but please take the latest pull from upstream whenever you update your PR. Your fork is 28 commits behind. Please take the latest pull and update your PR to get it merged.

**Comment by kumarishalini6** [comment]
ok

**Comment by kumarishalini6** [comment]
Done

**Comment by kumarishalini6** [comment]
@vaishali614 also assign me the another issue that i have created

**Pull request merged**